### PR TITLE
Completed string-manipulation-1 assignment. Bryan Valdovinos

### PR DESCRIPTION
**Pull request by bryanvaldovinos**
string manipulation 1

### CODE REVIEW

**Comment by bryanvaldovinos** [comment]
Screen Shot 2022-05-23 at 12 19 19 AM

File: string-manipulation-1/src/is-vowel.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+/* exported isVowel */
+function isVowel(char) {
+  if (char.length === 1) {
+    var vowels = 'aeiouAEIOU';
+    var isVowel = vowels.indexOf(char) >= 0;
```
Comment (codingcodymiller) [inline string-manipulation-1/src/is-vowel.js:5]: Try doing this without using `indexOf` 🤓

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

File: string-manipulation-1/src/is-vowel.js (hunk 1 above)
Comment (bryanvaldovinos) [reply string-manipulation-1/src/is-vowel.js]: done 👍

**Comment by bryanvaldovinos** [comment]
Screen Shot 2022-05-23 at 2 30 09 PM

**Pull request merged**